cmake/onnxruntime_providers_openvino.cmake

```diff
@@ -21,6 +21,10 @@
     message(FATAL_ERROR "OpenVINO 2024.0 and newer are supported. Please, use latest OpenVINO release")
   endif()
 
+  if(OpenVINO_VERSION VERSION_GREATER_EQUAL 2024.4)
+    add_definitions(-DUSE_DEVICE_MEMORY=1)
```

Comment:
USE_DEVICE_MEMORY is an OVEP specific macro?
it should be renamed.
is this macro even needed?  always build the device allocator code but use the OV version macros to determine whether to enable remote tensors support?

Reply:
yes, it's OVEP NPU device specific macro.
changed the name accordingly.